.coverage.ZBMAC-72080329e.68060.456692

```diff
@@ -0,0 +1 @@
n1/Desktop/koalas/databricks/koalas/version.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/mlflow.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/datetimes.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/generic.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/series.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/__init__.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/typedef.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/ml.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/indexes.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/utils.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/frame.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/exceptions.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/indexing.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/namespace.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/groupby.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/base.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_parquet.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_utils.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_stats.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_etl.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_series_conversion.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_namespace.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/__init__.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_reshape.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_series_datetime.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_indexing.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_dataframe_conversion.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_repr.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_indexes.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_groupby.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_series_string.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_series.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_csv.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/tests/test_dataframe.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/missing/series.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/missing/__init__.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/missing/indexes.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/missing/frame.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/missing/groupby.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/testing/__init__.py":[],"/Users/fengyurun1/Desktop/koalas/databricks/koalas/testing/utils.py":[]}}
```

Comment:
Could you remove this file?

Reply:
Hello ueshin. I have fix code with your kind guide. But i want to know why there always create coverge* file after `git push origin`.Is there some solution to avoid this?

Reply:
I guess you add it when you do `git add` command.

I added the file pattern in `.gitignore` in master branch. If you merge master into your branch, then it won't appear anymore.

Reply:
Btw, the files are created by `dev/pytest` for some reason.